src/org/ticket/view/CreateTicket.java

```diff
@@ -35,6 +72,49 @@ public CreateTicket() {
 
     @Override
     public void actionPerformed(ActionEvent e) {
+        Object source = e.getSource();
+
+        if(source.equals(bcreate)) {
+            disable();
+
+            TicketController tc = new TicketController();
+
+            Ticket.TicketPriority tp = Ticket.TicketPriority.LOW;
+            String tps = cbpriority.getSelectedItem().toString();
+
+            if(tps.equals("Média")) {
+                tp = Ticket.TicketPriority.MEDIUM;
```

Comment:
We will have to change this later but it is easy to do.

Reply:
This is what I was talking about. First compare indexes and you also have to change the value of the `priority` field inside the Ticket class.